<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1533</width>
    <height>1157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="console">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>801</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:white;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.875pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteMyselfFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>Delete File Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="defineFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Define File Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="writeFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Write File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bodEdit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>610</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>921600</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createMyselfFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>Create File Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readLocalFile">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>610</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Read Local File</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="deleteRegime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>delete mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearDirectory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>770</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>Clear Directory</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="console_2">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>10</y>
     <width>441</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:white;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.875pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkFileButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Code x02</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readFileButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Read File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showDirectory">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Directory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>770</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteLocalFile">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>610</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>Delete Local File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearConsole">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>610</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Console</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Fine Name (uint16_t)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkStatusButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Code x01</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="selectBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>640</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color :black;
font-size: 14px;
</string>
   </property>
   <property name="title">
    <string>Device Overview</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="vidLabel">
      <property name="text">
       <string>Vendor ID:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="locationLabel">
      <property name="text">
       <string>Location ID:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="pidLabel">
      <property name="text">
       <string>Product ID:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="manufacturerLabel">
      <property name="text">
       <string>Manufacturer Desc:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Chip Type:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="serialNumberLabel">
      <property name="text">
       <string>Serial Number:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="serialPortInfoListBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>610</y>
     <width>341</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
